ebpf-io/ebpf.io-website
# feat: develop summit 2023

**This PR:**
Adds three pages for the eBPF Summit 2023:
- Main summit page;
- Resources;
- Swag;

**Steps to test**

- Open the [Summit page](https://deploy-preview-423--ebpfio.netlify.app/summit-2023/)
- Open the [Resources page](https://deploy-preview-423--ebpfio.netlify.app/resources/)
- Open the [Swag page](https://deploy-preview-423--ebpfio.netlify.app/swag)
- Confirm that everything looks and works as expected
